Apps/W1/EDocumentsConnector/app/src/Continia/Implementation/EnvCleanupSubscr.Codeunit.al

```diff
@@ -0,0 +1,36 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.EServices.EDocumentConnector.Continia;
+using System.DataAdministration;
+using System.Environment;
+
+codeunit 6398 "Env. Cleanup Subscr."
+{
+    Access = Internal;
+
+    [EventSubscriber(ObjectType::Codeunit, Codeunit::"Environment Cleanup", 'OnClearCompanyConfig', '', false, false)]
```

Comment:
should we really clean the config ony any Environment Cleanup?
Maybe this should only be done, when a production environment is copied to a sandbox environment?